fix: serious grammar and spelling errors in docs

antews opened the pull request:
## Summary

In reviewing the project documentation, I have identified and corrected several errors and typos that were inadvertently made during its creation. These errors can lead to confusion or misunderstanding. 

## Details

Ensuring clarity and accuracy in documentation is critical for both users and developers. 

Closes #432 

ticket:
#432

OisinKyne commented on docs/advanced/quickstart-builder-api.mdx:
```diff
@@ -156,10 +157,10 @@ When your cluster is running, you should see if Charon is logging something like
 If you are using the [ultrasound relay](https://relay.ultrasound.money), you can enter your cluster's distributed validator public key(s) into their website, to confirm they also see the validator as correctly registered.
 
-You should check that your validator client's logs look healthy, and ensure that you haven't added a `fee-recipient` address that conflicts with what has been selected by your cluster in your cluster-lock file, as that may prevent your validator from producing a signature for the block when the opportunity arises. You should also confirm the same for all of the other peers in your cluster.
+You should check that your validator client's logs look healthy, and ensure that you haven't added a `fee-recipient` address that conflicts with what has been selected by your cluster in your `cluster-lock.json file`, as that may prevent your validator from producing a signature for the block when the opportunity arises. You should also confirm the same for all of the other peers in your cluster.
```
```suggestion
You should check that your validator client's logs look healthy, and ensure that you haven't added a `fee-recipient` address that conflicts with what has been selected by your cluster in your `cluster-lock.json` file, as that may prevent your validator from producing a signature for the block when the opportunity arises. You should also confirm the same for all of the other peers in your cluster.
```

OisinKyne commented on docs/advanced/quickstart-builder-api.mdx:
````diff
@@ -156,10 +157,10 @@ When your cluster is running, you should see if Charon is logging something like
 ```
 
-This indicates that your Charon node is successfully registering with the relay for a blinded block when the time comes.
+This indicates that your Charon node is successfully registering with the relay for a blind block when the time comes.
````
```suggestion
This indicates that your Charon node is successfully registering with the relay for a blinded block when the time comes.
```
They are called blinded blocks. https://ethereum.github.io/beacon-APIs/#/Validator/produceBlindedBlock

OisinKyne commented:
Hey i can't merge this due to conflicts, can you figure out does it need a rebase or what is up with it? :)

OisinKyne reviewed (commented):
Needs rebase

OisinKyne reviewed (changes requested):
Needs rebase

antews commented:
> Hey i can't merge this due to conflicts, can you figure out does it need a rebase or what is up with it? :)

Hi! @OisinKyne I've rebased and resolved all conflicts, now PR is ready for merge! Thanks!

antews commented:
> Hi! @OisinKyne I've rebased and resolved all conflicts, now PR is ready for merge! Thanks!

Hi! @OisinKyne  I've rebased and resolved all conflicts, now PR is ready for merge! Thanks!

OisinKyne commented:
Hey apologies. I was in a hurry to get these changes in before I tagged the next release such that they would show up outside the `next` section, so i implemented them in another PR. This PR just has two extra newlines left. Appreciate the follow up and rebase though, sorry for not coming back to this!

antews commented:
> Hey apologies. I was in a hurry to get these changes in before I tagged the next release such that they would show up outside the `next` section, so i implemented them in another PR. This PR just has two extra newlines left. Appreciate the follow up and rebase though, sorry for not coming back to this!

Now I'am won't be in the contributor list, right? @OisinKyne

antews commented:
> Hey apologies. I was in a hurry to get these changes in before I tagged the next release such that they would show up outside the `next` section, so i implemented them in another PR. This PR just has two extra newlines left. Appreciate the follow up and rebase though, sorry for not coming back to this!

?

slugmann321 commented:
@antews please only contribute if you actually want to see changes, not to get on a list. There is no "list" and being a contributor will not advantage you in any way. Thank you.
